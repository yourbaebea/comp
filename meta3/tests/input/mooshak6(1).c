int f(int);
char g(char);


int main(void) {
    f,p,argc;
    argc, f;
    f,g;
    0,('c', f);
}