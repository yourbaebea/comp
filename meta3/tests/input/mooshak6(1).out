Line 6, col 7: Unknown symbol p
Line 6, col 6: Operator , cannot be applied to types int(int), undef
Line 6, col 9: Unknown symbol argc
Line 6, col 8: Operator , cannot be applied to types undef, undef
Line 7, col 5: Unknown symbol argc
Line 7, col 9: Operator , cannot be applied to types undef, int(int)
Line 8, col 6: Operator , cannot be applied to types int(int), char(char)
Line 9, col 11: Operator , cannot be applied to types int, int(int)
Line 9, col 6: Operator , cannot be applied to types int, undef
===== Global Symbol Table =====
putchar	int(int)
getchar	int(void)
f	int(int)
g	char(char)
main	int(void)

===== Function main Symbol Table =====
return	int

Program
..FuncDeclaration
....Int
....Id(f)
....ParamList
......ParamDeclaration
........Int
..FuncDeclaration
....Char
....Id(g)
....ParamList
......ParamDeclaration
........Char
..FuncDefinition
....Int
....Id(main)
....ParamList
......ParamDeclaration
........Void
....FuncBody
......Comma - undef
........Comma - undef
..........Id(f) - int(int)
..........Id(p) - undef
........Id(argc) - undef
......Comma - int
........Id(argc) - undef
........Id(f) - int(int)
......Comma - char
........Id(f) - int(int)
........Id(g) - char(char)
......Comma - int
........IntLit(0) - int
........Comma - int
..........ChrLit('c') - int
..........Id(f) - int(int)
